CHANGELOG.md

```diff
@@ -79,6 +79,7 @@
 - Bugfix: Fixed character counter changing fonts after going over the limit. (#3422)
 - Bugfix: Fixed crash that could occur if the user opens/closes ChannelViews (e.g. EmotePopup, or Splits) then modifies the showLastMessageIndicator setting. (#3444)
 - Bugfix: Removed ability to reload emotes really fast (#3450)
+- Bugfix: Re-add date of build to the "About" page on nightly versions.(#3464)
```

Comment:
```suggestion
- Bugfix: Re-add date of build to the "About" page on nightly versions. (#3464)
```

Reply:
dunno if we want to specify `on cmake nightly versions`

Reply:
Might make sense, I will leave that decision to @pajlada though I think. Do you happen to know in what scenarios people rely on the QMake builds?